process_pr.py

```diff
@@ -1607,10 +1607,7 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
             elif len(commit.parents) > 1:
                 bot_cache["commits"][commit.sha]["files"] = []
 
-            if (
-                last_seen_commit_time
-                and commit.commit.committer.date.timestamp() > last_seen_commit_time
-            ):
+            if last_seen_commit_time is not None and commit.sha in new_commits:
```

Comment:
once calculation of `new_commits` is moved at the right place then no need to check for `last_seen_commit_time is not None`  (just `commit.sha in new_commits` should be enough)